src/js/profile/subcategory.js

```diff
@@ -55,7 +56,7 @@ export class Subcategory extends Component {
             for (const [title, indicatorData] of Object.entries(detail.indicators)) {
                 if (typeof indicatorData.data !== 'undefined') {
                     let isLast = index === lastIndex;
-                    let i = new Indicator(this, formattingConfig, wrapper, title, indicatorData, detail, isLast);
+                    let i = new IndicatorBlock(this, formattingConfig, wrapper, title, indicatorData, detail, isLast);
```

Comment:
Where does the MarkdownBlock get created?

Reply:
It would be created there, I'm not sure what the best approach would be to create Indicator or Markdown blocks, would an if statement be okay similar to how I had it before? Something like if indicatorData.html: and then the backend should set that